app/views/assignments/_assignment_form_options.html.erb

```diff
@@ -67,7 +67,7 @@
 
 <dl class="<%= view.form_class_for(:deadline) %>">
   <dt><label>Deadline <small>(optional)</small></label></dt>
-  <dd><%= f.text_field :deadline, class: 'jquery-datetimepicker form-control input-contrast input-block', value: DeadlineFormatHelper.convert(f.object.deadline) %></dd>
+  <dd><%= f.text_field :deadline, class: 'jquery-datetimepicker form-control input-contrast input-block', value: DeadlineFormatHelper.convert(f.object.deadline), :autocomplete => :off %></dd>
```

Comment:
We prefer to use the Ruby 1.9 hash literal syntax when your hash keys are symbols. ([style](https://github.com/rubocop-hq/ruby-style-guide#hash-literals))
```suggestion
  <dd><%= f.text_field :deadline, class: 'jquery-datetimepicker form-control input-contrast input-block', value: DeadlineFormatHelper.convert(f.object.deadline), autocomplete: :off %></dd>
```